tests/builder.rs

```diff
@@ -6,15 +6,22 @@ use orchard::{
     circuit::{ProvingKey, VerifyingKey},
     keys::{FullViewingKey, PreparedIncomingViewingKey, Scope, SpendAuthorizingKey, SpendingKey},
     note::{AssetBase, ExtractedNoteCommitment},
-    note_encryption_v3::OrchardDomainV3,
+    note_encryption::OrchardDomainBase,
+    orchard_flavor::OrchardZSA,
     tree::{MerkleHashOrchard, MerklePath},
     value::NoteValue,
     Anchor, Bundle, Note,
 };
 use rand::rngs::OsRng;
 use zcash_note_encryption_zsa::try_note_decryption;
 
-pub fn verify_bundle(bundle: &Bundle<Authorized, i64>, vk: &VerifyingKey, verify_proof: bool) {
+type OrchardDomainZSA = OrchardDomainBase<OrchardZSA>;
```

Comment:
In our discussion, we decided to keep the name `OrchardDomain` for now (along with `OrchardDomainBase`, `OrchardDomainVanilla`, and `OrchardDomainZSA`) because we have three traits - `OrchardDomain`, `OrchardCircuit`, and `OrchardHash`. This naming scheme with `Orchard` prefix looks consistent.